fastlane_bot/helpers/txhelpers.py

```diff
@@ -455,35 +417,29 @@ def build_tx(
 
         returns: the transaction to be submitted to the blockchain
         """
-        max_priority_fee = int(max_priority_fee)
-        base_gas_price = int(base_gas_price)
-        max_gas_price = base_gas_price + max_priority_fee
 
         if self.ConfigObj.NETWORK == self.ConfigObj.NETWORK_TENDERLY:
             self.wallet_address = self.ConfigObj.BINANCE14_WALLET_ADDRESS
             
-        # if "tenderly" in self.web3.provider.endpoint_uri:
-        #     print("Tenderly network detected: Manually setting maxFeePerFas and maxPriorityFeePerGas")
-        #     max_gas_price = 3
-        #     max_priority_fee = 3
-
         if self.ConfigObj.NETWORK in ["ethereum", "coinbase_base"]:
             tx_details = {
-                "type": "0x2",
```

Comment:
why the change string vs int? are both supported?

Reply:
`2` is cleaner than `"0x2"`.
No need to use strings where not necessary.